.circleci/config.yml

```diff
@@ -15,8 +15,8 @@ anchor_for_job_defaults: &job_defaults
 anchor_for_installing_dependencies: &install_dependencies
   name: Install dependencies
   command: |
-    source scripts/setup.sh || exit 1
-    source scripts/setup_gae.sh || exit 1
+    sudo pip install pyyaml
```

Comment:
Please get `python -m scripts.setup` to handle this as discussed.

Reply:
I am getting errors even after installing locally. This and the following two imports are required to be installed as sudo to run the tests in circleci. Also, it was here from before.

Reply:
Is this resolved? `install_third_party_libraries` should now take care of this, right?

Reply:
Still waiting here.

Reply:
Sorry this got lost in the long list of reviews. Done!